From 7d3eb179e6189dafcf0a89328128a37ed5e4a9c2 Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Thu, 29 Aug 2013 17:41:35 +0800
Subject: [PATCH 246/281] ENGR00277275 disable multi-user in mx6sl_evk

disable multi-user in mx6sl_evk as we found
multi-user has some problem with software agl.

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 arm2_6sl/overlay/frameworks/base/core/res/res/values/config.xml | 2 ++
 evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml  | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/arm2_6sl/overlay/frameworks/base/core/res/res/values/config.xml b/arm2_6sl/overlay/frameworks/base/core/res/res/values/config.xml
index a7177f0..79a5123 100755
--- a/arm2_6sl/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/arm2_6sl/overlay/frameworks/base/core/res/res/values/config.xml
@@ -98,5 +98,7 @@
     <add-resource type="string" name="config_gpsVendor" />
     <string name="config_gpsVendor">atheros</string>
 
+    <!--  Maximum number of supported users -->
+    <integer name="config_multiuserMaximumUsers">1</integer>
 
 </resources>
diff --git a/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml b/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml
index 56be38c..a585088 100644
--- a/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml
@@ -159,7 +159,7 @@
     <dimen name="config_viewConfigurationTouchSlop">24dp</dimen>
 
     <!--  Maximum number of supported users -->
-    <integer name="config_multiuserMaximumUsers">8</integer>
+    <integer name="config_multiuserMaximumUsers">1</integer>
 
     <!-- Whether a software navigation bar should be shown. NOTE: in the future this may be
          autodetected from the Configuration. -->
-- 
1.8.0

